rakelib/plugins-metadata.json

```
@@ -371,8 +371,8 @@
     "skip-list": false
   },
   "logstash-input-snmptrap": {
-    "default-plugins": true,
-    "skip-list": false
+    "default-plugins": false,
+    "skip-list": true
```

We still want to generate docs and overall treat this as a normal plugin of the Logstash ecosystem, we only want to not have it a default plugin:
```suggestion
    "skip-list": false
```